<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtBasedVideoEditor</class>
 <widget class="QMainWindow" name="QtBasedVideoEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>932</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QtBasedVideoEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" colspan="3">
       <widget class="QWidget" name="timelineWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timelineLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Timeline</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QSlider" name="timelineSlider">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="timelineZone" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QWidget" name="previewWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="previewLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Preview Window</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="playZone" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="playPauseButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Play</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSlider">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="videoWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="videoListLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Video List</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="videoList"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="materialWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="materialLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Material Library</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="materialList"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1106</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImportVideo"/>
    <addaction name="actionImportMaterial"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuRemove">
     <property name="title">
      <string>Remove</string>
     </property>
     <addaction name="actionRemoveVideo"/>
     <addaction name="actionRemoveMaterial"/>
    </widget>
    <addaction name="menuRemove"/>
    <addaction name="actionRenameVideo"/>
    <addaction name="actionCutVideo"/>
   </widget>
   <widget class="QMenu" name="menuPreview">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionPreview"/>
    <addaction name="actionPreviewResult"/>
   </widget>
   <widget class="QMenu" name="menuTimeline">
    <property name="title">
     <string>Timeline</string>
    </property>
    <widget class="QMenu" name="menuAdd_to_Timeline">
     <property name="title">
      <string>Add to Timeline</string>
     </property>
     <addaction name="actionAddVideoToTimeline"/>
     <addaction name="actionAddMaterialToTimeline"/>
    </widget>
    <addaction name="menuAdd_to_Timeline"/>
    <addaction name="actionDivideVideo"/>
    <addaction name="actionMergeVideos"/>
    <addaction name="actionPermuteVideosInTimeline"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTimeline"/>
   <addaction name="menuPreview"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionImportVideo">
   <property name="text">
    <string>Import Video</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export as ...</string>
   </property>
  </action>
  <action name="actionImportMaterial">
   <property name="text">
    <string>Import Material</string>
   </property>
  </action>
  <action name="actionCutVideo">
   <property name="text">
    <string>Cut Video...</string>
   </property>
  </action>
  <action name="actionDivideVideo">
   <property name="text">
    <string>Divide Video...</string>
   </property>
  </action>
  <action name="actionMergeVideos">
   <property name="text">
    <string>Merge Videos...</string>
   </property>
  </action>
  <action name="actionAddVideoToTimeline">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="actionAddMaterialToTimeline">
   <property name="text">
    <string>Material</string>
   </property>
  </action>
  <action name="actionRemoveVideo">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="actionRemoveMaterial">
   <property name="text">
    <string>Material</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="text">
    <string>Preview</string>
   </property>
  </action>
  <action name="actionPreviewResult">
   <property name="text">
    <string>Preview Result</string>
   </property>
  </action>
  <action name="actionRenameVideo">
   <property name="text">
    <string>Rename Video...</string>
   </property>
  </action>
  <action name="actionPermuteVideosInTimeline">
   <property name="text">
    <string>Permute Videos in Timeline</string>
   </property>
  </action>
  <action name="actionAddNewTextToTimeline">
   <property name="text">
    <string>New Text</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
